tests/unit/test_transforms.py

```diff
@@ -1426,3 +1429,143 @@ def test_transform_annotation_type_label_with_nan(self):
+                    [
+                        (
+                            "Age",
+                            float,
+                            {10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 70.0, 80.0, 90.0, 100.0},
+                        ),
+                        ("Review Text", str),
+                        ("Rating", CategoricalDtype(), {1.0, 2.0, 3.0, 4.0, 5.0}),
+                        (
+                            "Division Name",
+                            CategoricalDtype(),
+                            {"General", "General Petite", "Initmates"},
+                        ),
+                    ]
+                )
+            },
+            media_type=TableRow,
+        )
+
+        tabular_refined_path = osp.join(
+            get_test_asset_path("tabular_dataset"), "women-clothing", "women_clothing_refined.csv"
+        )
+        table = Table.from_csv(tabular_refined_path)
+        self.refined_tabular_dataset = Dataset.from_iterable(
+            [
+                DatasetItem(
+                    id=i,
+                    subset="train",
+                    media=TableRow(table=table, index=i),
+                    annotations=[
+                        Tabular(
+                            values={
+                                "Rating": table.data["Rating"][i],
+                                "Age": table.data["Age"][i],
+                                "Title": table.data["Title"][i],
+                                "Review Text": table.data["Review Text"][i],
+                                "Division Name": table.data["Division Name"][i],
+                            }
+                        )
+                    ],
+                )
+                for i in range(0, 10)
+            ],
+            categories={
+                AnnotationType.tabular: TabularCategories.from_iterable(
+                    [
+                        (
+                            "Age",
+                            float,
+                            {10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 70.0, 80.0, 90.0, 100.0},
+                        ),
+                        ("Review Text", str),
+                        ("Rating", CategoricalDtype(), {1.0, 2.0, 3.0, 4.0, 5.0}),
+                        (
+                            "Division Name",
+                            CategoricalDtype(),
+                            {"General", "General Petite", "Initmates"},
+                        ),
+                    ]
+                )
+            },
+            media_type=TableRow,
+        )
+
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_remove_unneccessary_char(self):
+        example_text = "This is a test 😊! Check out https://example.com for more <b>details</b> about this text. Enjoy!!!"
+        cleaned_text = "test check details text enjoy"
+
+        with self.subTest("with None"):
+            self.assertIsNone(transforms.Clean.remove_unneccessary_char(None))
```

Comment:
Sorry that I am not sure about the way to use `transforms` in Datumaro.
As far as I know, Datumaro mostly uses `transforms` like `dataset.transforms("clean", xxx)`.
Could you clarify this by providing the way to use clean transform in a general way or finding some examples like yours (transforms.Clean.xxx).

Reply:
I fixed this in [cca8b29](https://github.com/openvinotoolkit/datumaro/pull/1520/commits/cca8b29229e1b86bec4745125d20ca3bd03fad61) for other cases. 
In this scenario, I've utilized the aforementioned approach to employ `transform` for testing the static functions of the class. If there are alternative methods to test these functions, please inform me, and I'll make the necessary updates.
Additionally, other tests for 'transform' adopt the usage pattern of `transform.Clean(dataset),` and thus, I've followed suit. Although there are variations in results between these methods, such as obtaining a class or dataset, it doesn't affect the functionality testing of the dataset. This is because we can employ `transform_item` while comparing datasets through iterators. 
Moreover, I've utilized the approach of `dataset.transforms("clean")` for real-case scenarios and restricted the use of `transform.Clean(dataset)` solely for testing, aligning with other tests. If any adjustments are needed, please advise accordingly

Reply:
Could you please provide an example of performing `remove_unneccessary_char` in `dataset.transforms("clean")` way?